internal/conf/configuration.go

```diff
@@ -205,6 +205,7 @@ type SmsProviderConfiguration struct {
 }
 
 type TwilioProviderConfiguration struct {
+	VerifyEnabled     bool   `json:"verify_enabled" split_words:"true"`
 	AccountSid        string `json:"account_sid" split_words:"true"`
 	AuthToken         string `json:"auth_token" split_words:"true"`
 	MessageServiceSid string `json:"message_service_sid" split_words:"true"`
```

Comment:
In my opinion, you should extract Twilio Verifty to be a _new_ provider. So register a new struct `TwilioVerifyProvider` etc. Don't re-use the existing one, which you can rename `TwilioProgrammableMessagingProvider` or such.